src/pybind/mgr/dashboard/frontend/src/app/shared/datatable/table/table.component.scss

```diff
@@ -10,6 +10,7 @@
 
 .dataTables_wrapper {
   margin-bottom: 25px;
+  max-width: 99.9%;
```

Comment:
Why 99.9% and not 100%? Maybe worth adding some comment explaining such a tricky value.

Reply:
It was a hack to prevent [this issue ](https://github.com/ceph/ceph/pull/40618#pullrequestreview-629010639). I couldn't find an alternate for this.

Reply:
Yes, please. So the issue was already there... 🤔

Reply:
After bootstrap 8, this seems to be happening. I am pretty sure its related to our implementation. I've tried debugging it a lot but couldn't find the cause. Added a comment.